ansible/roles/kafka/tasks/clean.yml

```diff
@@ -3,16 +3,8 @@
 
 - name: remove kafka
```

Comment:
just recognized an issue with rollout into existing environments and while testing it on PGs. While we do transition we should keep a clean step to remove `kafka` container. Since the new clean task would only clean `kafkaX` containers but leave `kafka` containers which could result in a port conflict.

Proposal: pls keep a second `remove kafka` task in here. we could remove after some time. (same for zookeeper removal below)